scripts/drtprod

```diff
@@ -112,13 +112,79 @@ EOF"
     roachprod ssh ${cluster} -- "sudo tee /etc/datadog-agent/conf.d/cockroachdb.d/conf.yaml > /dev/null << EOF
 ---
 init_config:
+
 instances:
 - openmetrics_endpoint: http://localhost:26258/_status/vars
   tls_verify: false
+  service: drt-cockroachdb
 EOF"
 
     roachprod ssh ${cluster} -- 'sudo systemctl enable datadog-agent && sudo systemctl restart datadog-agent'
 
+    # Single quotes around the shell expansion cause it to be processed on the
```

Comment:
Is the plan to move this into a `roachprod install` command? It seems a _tad_ weird to me that we have core roachprod changes to enable fluent logging, but only the steps to _install_ fluent to log _to_ in the the drtprod wrapper, if I'm reading this correctly?

Reply:
I'm still figuring out the API boundaries here, but I think I can move metrics and logs into `roachprod install` and then update `drtprod` to call the necessary `roachprod install` commands and restart the nodes.

I added the `--enable-fluent-sink` option to `roachprod` so that only DRT clusters would have the `fluent-servers` logging configuration. Otherwise every cluster would log errors that it couldn't send logs to a non-existent Fluent Bit server.

I'm happy to take the time to do this refactor now or in a subsequent pull request. Any preference?

Reply:
later works for me

Reply:
as far as API boundaries: I'm 100% good with doing dRT-specific stuff in `drtprod`, rather than in `roachprod`. But the one reason I asked here was we're extending core roachprod to add the flag to tell CRDB to use fluent, which, sure, yeah, fine. But someone, on seeing that, is likely to go "okay, so I can tell it to use fluent, so how do I install fluent?" and then the answer for _that_ isn't in roachprod which is a little bit weird. But little bit weird is still better than no support for fluent -- either starting with it or installing it or both -- at all, so I don't mean to let perfect be enemy of good here.